game-core/src/main/java/games/strategy/engine/framework/startup/ui/editors/ForumPosterEditor.java

```diff
@@ -1,296 +0,0 @@
-package games.strategy.engine.framework.startup.ui.editors;
-
-import com.google.common.annotations.VisibleForTesting;
-import com.google.common.base.Preconditions;
-import games.strategy.engine.ClientContext;
-import games.strategy.engine.data.properties.GameProperties;
-import games.strategy.engine.framework.GameRunner;
-import games.strategy.engine.posted.game.pbf.IForumPoster;
-import games.strategy.engine.posted.game.pbf.NodeBbForumPoster;
-import java.awt.Graphics;
-import java.awt.GridBagConstraints;
-import java.awt.Insets;
-import java.awt.image.BufferedImage;
-import java.io.File;
-import java.io.IOException;
-import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.ExecutionException;
-import java.util.logging.Level;
-import javax.imageio.ImageIO;
-import javax.swing.JButton;
-import javax.swing.JCheckBox;
-import javax.swing.JComboBox;
-import javax.swing.JLabel;
-import javax.swing.JOptionPane;
-import javax.swing.JTextField;
-import javax.swing.SwingUtilities;
-import lombok.extern.java.Log;
-import org.triplea.java.TimeManager;
-import org.triplea.swing.DocumentListenerBuilder;
-import org.triplea.swing.ProgressWindow;
-
-/** A class for selecting which Forum poster to use. */
-@Log
-public class ForumPosterEditor extends EditorPanel {
```

Comment:
Most of the code in this class is moved but not modified. There are enough changes that git thinks it is a new file post-move, see individual commits for better diff'ing of updates.